src/cljs/athens/electron.cljs

```diff
@@ -323,6 +325,40 @@
   ;; open or create a new starter db
 
   ;; Watch filesystem, e.g. in case db is updated via Dropbox sync
+  
+    
+  ;; Update index.transit    
+  (defn write-db-index [file filepath]
+    (let [w  (.. fs (writeFile filepath file (fn [e] (dispatch [:boot/desktop]))))])
+  )  
+  
+  (defn open-dialog-index
+    "Allow user to open a Backup file."
+    [filepath]
+    (js/alert "Your data file is corrupted or incorrect, please select a backup file.")
```

Comment:
if the user does not have a backup file, looks to me like this leaves them in an infinite loop if they try to close the dialog. could we have a confirm dialog maybe? like "do you want to load a backup, or start a new database?".

Reply:
should i add an extra  view like ns athens.views.alerts
with the only two options  
Load a Backup file or Create a new Db